docs/datasets/index.rst

```diff
@@ -52,10 +52,83 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
+To analyse multiple runs, you can either stack the datasets together, or perform
+a joint fit across multiple datasets.
 
-Getting Started
-===============
+Stacking Multiple Datasets
+==========================
 
+Stacking datasets implies that the counts, background and reduced IRFs from all the
+runs are binned together to get one final dataset for which a likelihood is
+computed during the fit.
+
+Counts, background and exposure (lying outside the masked regions are simply summed),
+while for energy dispersion and point spread function an avergae is computed after
+weighing by the exposure.
+
+For the model evaluation, an important factor that needs to be accounted for is
+that the energy threshold changes between obseravtions.
+To ensure that the npred (ie, the predicted number of counts) on the stacked
+dataset is the sum expected by stacking the npred of the individual runs,
+a `~gammapy.irf.EDispersionMap` is used, which contains the
+The mask_safe from each dataset is applied on the respective reconstructed energy axis
+of the energy dispersion matrix, and the masked matrices are combined.
+Values lying outside the safe mask of each dataset are lost.
+
+Stacking of multiple datasets is implemented as follows.
+Here, :math:`k` denotes a bin in reconstructed energy,
+:math:`l` a bin in true energy and
+:math:`j` is the dataset number
+
+
+================= ================================== ==================================================================================
+Dataset attribute Behaviour                          Implementation
+================= ================================== ==================================================================================
+
+``livetime``.        Sum of individual livetimes         :math:`\overline{t} = \sum_j t_j`
+
+``mask_safe``        Pixels added with `OR` operation     :math:`\overline{\epsilon_k} = \sum_{j} \epsilon_{jk}`
```

Comment:
In the implementation one could add the definition of: the mask is true if the pixel is included in the safe data range.